pkg/internal/util/cli/spinner.go

```diff
@@ -60,7 +63,11 @@ type Spinner struct {
 var _ io.Writer = &Spinner{}
 
 // NewSpinner initializes and returns a new Spinner that will write to w
+// NOTE: w should be os.Stderr or similar, and it should be a Terminal
 func NewSpinner(w io.Writer) *Spinner {
+	if v, ok := w.(*os.File); ok {
+		w = colorable.NewColorable(v)
```

Comment:
NOTE: on not-Windows this just returns `v`. On Windows it wraps the writer with some fun magic to handle ANSI codes :-)